### DISCUSSION TITLE
Giving a commit range to Fogit

### DISCUSSION THREAD

**Discussion by marlun** [rbong/vim-flog#119] [Q&A] [answered]
After running `git fetch origin`, with tig I can run `tig ..origin` to see only the commits which is new and will be included if I merge. I can't find how to do the same with Fogit, is there another way to do it?

**Comment by rbong** [comment, marked as answer]
When you are using `tig ..origin` it is the same as `git log ..origin`. The first positional argument to `git log` and `tig` is the Git revision descriptor.

Flog does not support positional arguments, but you can pass the revision via the `-rev` argument.

Therefore, you can do this by running `:Flog -rev=..origin`

**Comment by rbong** [comment]
Re-opening because closed discussions aren't as visible.
